manifests/base/controller-manager/controller-config.yaml

```diff
@@ -26,6 +26,8 @@ data:
           maxBytes: -1
           replicas: 3
           duplicates: 300s
+          retention: LimitsPolicy
```

Comment:
Using Nats original config and adding comments like below?

```
            # 0: Limits, 1: Interest, 2: WorkQueue
            retention: 0
            # 0: DiscardOld, 1: DiscardNew
            discard: 0
```

Reply:
done thank you!
Is the E2E Tests (kafka) `expected sensor pod log contains 'Successfully processed trigger 'log-trigger''` failure known and expected?